projects/components/src/overlay/sheet/sheet.ts

```diff
@@ -11,7 +11,7 @@ export interface SheetOverlayConfig<TData = unknown> extends OverlayConfig {
   closeOnEscapeKey?: boolean;
   closeOnNavigation?: boolean;
   attachedTriggerTemplate?: TemplateRef<unknown>;
-  pageNavParams?: ExternalNavigationParams;
+  pageNavParams?: NavigationParams;
```

Comment:
We shouldn't change this without adding appropriate support in the sheet overlay. Currently the icon represents a new tab behavior. We shouldn't link that to internal nav.